learnify/mobile/lib/features/auth/login/view-model/login_view_model.dart

```diff
@@ -71,6 +71,16 @@ class LoginViewModel extends BaseViewModel {
   }
 
   Future<String?> _loginRequest() async {
+    // TODO:
+    const User user = User(id: '2313');
+    await localManager.setModel(user, StorageKeys.user);
+    await localManager.setString(StorageKeys.accessToken, 'toke');
+    await localManager.setInt(StorageKeys.accessTokenExpires,
+        DateTime.now().add(const Duration(days: 10)).millisecondsSinceEpoch);
+    await navigationManager.navigateToPageClear(
```

Comment:
I understand the addition of token expiration, but isn't the rest repeating line 104+ with no changes and no deletion?